www/front_src/src/Resources/Graph/Performance/Legend/index.tsx

```diff
@@ -35,25 +40,20 @@ const useStyles = makeStyles<Theme, { panelWidth: number }>((theme) => ({
   hidden: {
     color: theme.palette.text.disabled,
   },
-  icon: {
-    borderRadius: '50%',
-    height: 9,
-    marginRight: theme.spacing(1),
-    width: 9,
-  },
   item: {
     display: 'grid',
     gridTemplateColumns: 'min-content minmax(50px, 1fr)',
-    margin: theme.spacing(0, 1, 1, 1),
+    marginBottom: theme.spacing(1),
   },
-  items: {
+  items: ({ limitLegendRows }) => ({
     display: 'grid',
     gridTemplateColumns: 'repeat(auto-fit, minmax(150px, 1fr))',
     justifyContent: 'center',
-    maxHeight: theme.spacing(16),
+    marginLeft: theme.spacing(0.5),
+    maxHeight: limitLegendRows ? theme.spacing(16) : '25vh',
```

Comment:
What is 25vh based on? shouldn't we simply take the remaining space?

Reply:
Oups, I missed to change that